src/commands/veloce/debug/logs.ts

```diff
@@ -51,20 +51,28 @@ export default class Org extends SfdxCommand {
     const authorization = Buffer.from(JSON.stringify(params)).toString('base64')
     const headers = {
       'dev-token': debugSession.token,
-      'Authorization': authorization,
-      'Content-Type': 'application/json'
+      'Authorization': authorization
     }
     const backendUrl: string | undefined = debugSession.backendUrl
 
+    this.ux.log('Following the backend logs...')
+
+    await new Promise(() => void this.callToGetLogs(backendUrl, headers));
+
+    return {}
+  }
+
+  private async callToGetLogs(backendUrl: string | undefined,
+                              headers: { Authorization: string; 'dev-token': any }): Promise<void> {
     try {
-      await axios.post(`${backendUrl}/services/dev-override/logs`, {}, {
-        headers
-      })
-    } catch ({ data }) {
-      this.ux.log(`Failed to list debug sessions: ${data as string}`)
-      return {}
+      const response = await axios.get(`${backendUrl}/services/dev-override/logs`, {headers});
+      if (response.data !== '') {
+        this.ux.log(response.data)
+      }
+    } catch (e) {
+      this.ux.log(`Failed to get logs: ${e as string}`)
```

Comment:
this example:
 catch (error) {
          const err = error as AxiosError
          this.ux.log(`Failed to deploy ${group.name}: ${err?.response?.data} code ${err?.response?.status}`);
        }
lets agree on catches to not be just "as string".

Reply:
I applied your suggestion. Thanks.